params/config.go

```diff
@@ -543,10 +526,16 @@ type BlobConfig struct {
 
 // BlobScheduleConfig determines target and max number of blobs allow per fork.
 type BlobScheduleConfig struct {
-	Cancun *BlobConfig `json:"cancun,omitempty"`
-	Prague *BlobConfig `json:"prague,omitempty"`
-	Osaka  *BlobConfig `json:"osaka,omitempty"`
-	Verkle *BlobConfig `json:"verkle,omitempty"`
+	Cancun  *BlobConfig `json:"cancun,omitempty"`
+	Prague  *BlobConfig `json:"prague,omitempty"`
+	Prague1 *BlobConfig `json:"prague1,omitempty"`
+	Osaka   *BlobConfig `json:"osaka,omitempty"`
+	Verkle  *BlobConfig `json:"verkle,omitempty"`
+}
+
+// IsBerachain returns whether the node is a berachain node or not.
```

Comment:
TODO: add validation to startup to fail for running bepolia or berachain is IsBerachain returns false